File: AmazingBooks-API/Entities/OrderLine.cs
using System;
using System.Collections.Generic;

namespace AmazingBooks_API.Entities;

public partial class OrderLine
{
    public int Id { get; set; }

    public decimal Amount { get; set; }

    public int Quantity { get; set; }

    public decimal? Weight { get; set; }

    public int FkorderId { get; set; }

    public int FkbookId { get; set; }

    public virtual Book Fkbook { get; set; } = null!;

    public virtual Order Fkorder { get; set; } = null!;
}

File: AmazingBooks-UI/src/app/model/orderLine.ts
import { Book } from './book';
import { Order } from './order';

export interface OrderLine {
  id: number;
  amount: number;
  quantity: number;
  weight?: number;
  fkorderId?: number;
  fkbookId: number;
  fkbook?: Book;
  fkorder?: Order;
}
